discovery-infra/test_infra/consts/env_defaults.py

```diff
@@ -27,4 +27,6 @@
 DEFAULT_IMAGE_FOLDER: Path = Path(consts.IMAGE_FOLDER)
 DEFAULT_IMAGE_FILENAME: str = "installer-image.iso"
 DEFAULT_NETWORK_TYPE: str = consts.NetworkType.OpenShiftSDN
+DEFAULT_DISK_ENCRYPTION_MODE: str = consts.DiskEncryptionMode.TPM_VERSION_2
```

Comment:
Does that mean we will enable TPM in bios even when we run a non-encrypted installation flow?